specifications/api/swagger.yaml

```
@@ -410,6 +410,12 @@ x-transactionDirection: &transactionDirection
     - outgoing
     - incoming
 
+x-addressesInputs: &addressesInputs
+  description: A list of addresses inputs
```

```suggestion
x-addresses: &addresses
  description: A list of addresses
```

I think the "inputs" suffix is unnecessary here.

---

fixed in https://github.com/input-output-hk/cardano-wallet/pull/1664/commits/cf2b9e58c71fd7e80f8a9e1b9073348dfba3a67e